Astar Update: 2023-01-substrate-newsletter.md (#8)
substrate-developer-hub/newsletter: draft/2023-01-substrate-newsletter.md

## TL;DR and important announcements

## 🔦 Community highlights

> Astar launches a [**new website**](https://medium.com/astar-network/astar-2023-a-new-website-and-starmap-8f97dc9c0f1c) and introduces a Starmap to follow 2023 developer updates. The website is now focused on Community, Developers, and Japan initiatives!
> 
> Read [Astar's 2022 Recap + Building into 2023](https://medium.com/astar-network/welcome-to-the-future-f69f1324af1c) for all major updates.

Reviewer: ```suggestion
Astar introduces a [Starmap](https://astar.network/starmap/) along with their new website to follow 2023 developer updates. Read [Astar's 2022 Recap + Building into 2023](https://medium.com/astar-network/welcome-to-the-future-f69f1324af1c) for all major upcoming updates.
```

Reviewer: I may drop the first phrase about the site and replace it with:

**Astar’s XVM SDK Public Testnet is live!** The dev team explains the code, [here](https://medium.com/astar-network/cross-virtual-machine-creating-a-portal-to-the-future-of-smart-contracts-a96c6d2f79b8). Create an ink! smart contract that controls ERC20, ERC721, and more assets with [Ink! XVM SDK](https://github.com/AstarNetwork/ink-xvm-sdk).

Makes it more relevant to general builder community, imo FYI. I also only put key updates from Substrate/Polkadot/Cumulus in the Technical Updates section. So for community contribution, the goal is to populate the "upcoming events" and "community highlights" sections. At least for now - open to suggestions!

Author: Great suggestions. I think we should continue with your plan for now.

If I see areas for improvement, then I will suggest.
